applications/Chat/evaluate/evaluator.py

```diff
@@ -74,17 +80,22 @@ def switch(metric):
             for metric in category_metrics:
                 self.automatic_metric_stats[category].update(switch(metric=metric))
 
-        # gpt35 evaluation
+        # gpt evaluation
         for category in self.params:
-            category_metrics = self.params[category]["GPT-3.5"]
+            if len(answers_per_category[category]) == 0:
+                print(f"Category {category} specified in your config doesn't have corresponding answers!")
+                continue
+
+            category_metrics = self.params[category]["GPT"]
 
             prompt = self.gpt_evaluation_prompt.get(category, None)
             if prompt is None:
                 print(f"No prompt for category {category}! Use prompt for category general now.")
                 prompt = self.gpt_evaluation_prompt["general"]
 
-            self.gpt35_evaluation_results[category] = gpt_evaluate.gpt35_evaluate(answers_per_category[category],
-                                                                                  prompt, category_metrics, category)
+            self.gpt_evaluation_results[category] = gpt_evaluate.gpt_evaluate(answers_per_category[category], prompt,
```

Comment:
`gpt_evaluate.gpt_evaluate` is a bit confusing. Please rename it to `gpt_evaluate.evaluate` or other function names that shows the differences.